app/mailers/asset_notification.rb

```diff
@@ -6,6 +6,7 @@ def upload_notification(asset, email, _sent_at = Time.now)
     @description = asset.description
     @name = asset.user.name
     @user = asset.user
+    @title = asset.title.present? ? asset.title : "new track"
```

Comment:
I just tested uploading a new file, and it seems to grab `filename` and save it as `title` by default. Should I just leave this like this?  I wanted to add a placeholder in case if title is not available in some older songs... going to unpack the db dump and check actually :)